<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <!--Launch file for sim-->

  <node name="fake_angle_finder" pkg="prop_mapper" type="fake_angle_finder" output="log">
    <rosparam file="$(find prop_follower)/config/params.yaml" command="load" />
  </node>

  <node name="distance_finder" pkg="prop_follower" type="coord_finder" output="log">
    <rosparam file="$(find prop_mapper)/config/params.yaml" command="load" />
    <remap from="/scan" to="/laser/scan"/>
  </node>

  <node name="coord_finder" pkg="prop_follower" type="coord_finder" output="log">
    <rosparam file="$(find prop_follower)/config/params.yaml" command="load" />
  </node>

</launch>